sdk/identity/identity/test/msalTestUtils.ts

```diff
@@ -14,7 +14,11 @@ import { OperationTracingOptions, setSpan, context as otContext } from "@azure/c
 import { SpanGraph, setTracer } from "@azure/test-utils";
 import { MsalBaseUtilities } from "../src/msal/utils";
 import * as dotenv from "dotenv";
-dotenv.config();
+import { isNode } from "@azure/core-util";
+
+if (isNode) {
+  dotenv.config();
```

Comment:
The recorder should initialize this for you.

Reply:
Duh! Thank you!